VueJS/src/components/AddListItem.vue

```diff
@@ -20,7 +20,6 @@ export default {
   },
   computed: {
     listeners() {
-      /*  https://codesandbox.io/s/o29j95wx9 */
```

Comment:
For the future, I think it is better to keep links like that as we need to give credit to the people whose code we used to implement the functionality.